cli/test/test_scaffold.py

```diff
@@ -0,0 +1,29 @@
+from click.testing import CliRunner
+
+import popper.commands.cmd_scaffold as scaffold
+import popper.commands.cmd_run as run
+from test_common import PopperTest
+
+import unittest
+import os
+
+
+class TestScaffold(PopperTest):
+
+    def test_scaffold(self):
+
+        repo = self.mk_repo()
+        runner = CliRunner()
+        file_loc = repo.working_dir+'/wf.yml'
+
+        result = runner.invoke(scaffold.cli, ['-f', file_loc])
+
+        self.assertEqual(result.exit_code, 0)
+        self.assertTrue(os.path.isfile(file_loc))
```

Comment:
we can also check the content of the file to ensure that scaffolding did what it should. To do this, we can create a `Workflow` object from the file, and then compare it as shown [here](https://github.com/systemslab/popper/blob/master/cli/test/test_parser.py#L246).